scheduler/src/cook/mesos/api.clj

```diff
@@ -194,6 +194,15 @@
   {:name (s/constrained s/Str non-empty-max-128-characters-and-alphanum?)
    :version (s/constrained s/Str non-empty-max-128-characters-and-alphanum?)})
 
+(def Constraint
+  "Schema for user defined job host constraint"
+  [(s/one NonEmptyString "attribute")
+   (s/one (s/pred #(contains? (set (map (comp name util/without-ns) constraint-operators))
```

Comment:
Oh wow, yes. I didn't know name would strip the namespace as well. Will change now